src/components/elementView/detailedView/assetView/assetView.tsx

```diff
@@ -202,73 +222,80 @@ export class AssetView extends React.PureComponent<
                 cancelButtonText="Cancel"
                 confirmButtonText="Delete"
                 intent="danger"
+                icon="warning-sign"
                 isOpen={this.state.isDeleteOpen}
                 onCancel={this.handleDeleteCancel}
                 onConfirm={this.handleDelete}
               >
-                <p>Are you sure you want to delete?</p>
+                <p>Are you sure you want to <b>delete</b> this asset?  Unless it was created in error, consider <b>decommissioning</b> instead.</p>
```

Comment:
i dont know why but i find this a hilarious edit

Reply:
Thank you.